vyper/codegen/stmt.py

```diff
@@ -51,10 +44,12 @@ def __init__(self, node: vy_ast.VyperNode, context: Context) -> None:
         self.ir_node.annotation = self.stmt.get("node_source_code")
         self.ir_node.source_pos = getpos(self.stmt)
 
-    def parse_Expr(self):
-        # TODO: follow analysis modules and dispatch down to expr.py
+    def parse_Await(self):
```

Comment:
Should this be `Extcall`?
```suggestion
    def parse_Extcall(self):
```

Reply:
whoops, that's a dead function. ExtCall is a few lines down.

Reply:
Oh I see, there is no `parse_Extcall` in `stmt.py`, but the `Extcall` is wrapped in an `Expr` node so it would be handled by `parse_Expr` in `expr.py`.

Reply:
yep, Call is handled the same way now

Reply:
yep, Call is handled the same way now